core/services/feeds/service.go

```diff
@@ -171,7 +174,7 @@ func (s *service) UpdateFeedsManager(ctx context.Context, mgr FeedsManager) erro
 	}
 
 	if err = s.connMgr.Disconnect(mgr.ID); err != nil {
-		return errors.Wrap(err, "could not restart the connection")
+		logger.Info("[Feeds] Feeds Manager not connected, attempting to connect")
```

Comment:
An error only occurs here if the connection was not active, so we want to continue with the connection attempt